python/arcticdb/toolbox/query_stats.py

```diff
@@ -0,0 +1,89 @@
+from contextlib import contextmanager
+import numpy as np
+
+from arcticdb_ext.tools import QueryStats
+from arcticdb_ext.tools.QueryStats import StatsGroupName, StatsName
+
+class QueryStatsTool:
```

Comment:
Like I said in the other comment, we don't need a class for this. I think these can all be free functions defined in this module